docs/src/main/tut/faq.md

```diff
@@ -134,6 +135,63 @@ Cats defines a wealth of type classes and type class instances. For a number of
 
 More about the history of machinist and how it works can be discovered at the [project page](https://github.com/typelevel/machinist), or [this article on the typelevel blog](http://typelevel.org/blog/2013/10/13/spires-ops-macros.html).
 
+## <a id="tailrecm" href="#tailrecm"></a>What is `tailRecM`?
+
+The `FlatMap` type class has a `tailRecM` method with the following signature:
+
+```tut:fail:silent
```

Comment:
Should we just use `scala` modifier here since it's only failing because it's not implemented?